docs/api/products.rst

```diff
@@ -36,150 +188,428 @@ Response will contain a paginated list of products.
+                "description": "Et qui neque at sit voluptate sint omnis. Quos assumenda magni eos nemo qui accusamus.",
+                "short_description": "Molestiae quaerat in voluptate."
             }
         },
-        "_embedded":{
-            "items":[
-                {
-                    "created_at": "2014-11-26T23:00:20+0000",
-                    "description": "Facere ipsum id eveniet rem omnis et. Totam vero eos eveniet nihil sint. Labore occaecati qui placeat fugit.",
-                    "id": 2173,
-                    "name": "T-Shirt \"ipsam\"",
-                    "code": "T-SHIRT-IPSAM",
-                    "short_description": "Aut rerum quasi neque.",
-                    "updated_at": "2014-11-26T23:00:20+0000"
-                }
-            ]
+        "product_taxons": [
+            {
+                "id": 2,
+                "position": 1
+            }
+        ],
+        "main_taxon": {
+            "name": "Mugs",
+            "id": 2,
+            "code": "mugs",
+            "children": []
+        },
+        "reviews": [
+            {
+                "id": 41,
+                "title": "Nice",
+                "rating": 2,
+                "comment": "Nice",
+                "author": {
+                    "id": 22,
+                    "email": "banana@exmp.com",
+                    "email_canonical": "banana@exmp.com",
+                    "gender": "u"
+                },
+                "status": "new",
+                "created_at": "2017-01-18T11:15:44+0100",
+                "updated_at": "2017-01-18T11:15:45+0100"
+            }
+        ],
+        "average_rating": 2,
+        "images": [
+            {
+                "id": 3,
+                "code": "main",
+                "path": "af/ae/88f740736b8b79696513a5fe9c31.jpeg"
+            }
+        ],
+        "_links": {
+            "self": {
+                "href": "/api/v1/products/2"
+            }
         }
     }
 
-Getting a single product
-------------------------
+Creating Product
+----------------
 
-You can view a single product by executing the following request:
+Definition
+..........
 
 .. code-block:: text
 
-    GET /api/v1/products/2173
+    POST /api/v1/products/
+
++---------------+----------------+--------------------------------------------------------+
+| Parameter     | Parameter type | Description                                            |
++===============+================+========================================================+
+| Authorization | header         | Token received during authentication                   |
++---------------+----------------+--------------------------------------------------------+
+| code          | request        | **(unique)** Product identifier                        |
++---------------+----------------+--------------------------------------------------------+
+| translations  | request        | Contains product name and its **(unique)** slug        |
```

Comment:
Check how we used to define parameters [here](http://docs.sylius.org/en/latest/api/checkouts.html#parameters)
so it should be:
```
translations['locale_code']['name'] | request | Name of the product
translations['locale_code']['slug'] | request | **(unique)** Slug
```